dsew_community_profile/delphi_dsew_community_profile/pull.py

```diff
@@ -303,6 +303,8 @@ def retain_header(header):
             # exclude "People with a completed primary series as % of adult population"
             header.find("%") < 0,
             # exclude "People with a completed primary series - ages 65+"
+            header.find(" age") < 0,
```

Comment:
question/praise: did the test reveal this missed corner case?

if so, nicely done! i mean nicely done regardless, but it's particularly good to find this stuff out in tests rather than get a surprise in prod.